.env

```diff
@@ -50,4 +50,5 @@ MESSENGER_TRANSPORT_EXPORT_DSN=amqp://guest:guest@localhost:5672/%2f/exports
 MESSENGER_TRANSPORT_DOMAIN_DSN=amqp://guest:guest@rabbit:5672/%2f/domain
 MESSENGER_TRANSPORT_CHANNEL_DSN=amqp://guest:guest@localhost:5672/%2f/channel
 MESSENGER_TRANSPORT_SEGMENT_DSN=amqp://guest:guest@localhost:5672/%2f/segment
+MESSENGER_TRANSPORT_COMPLETENESS_DSN=amqp://guest:guest@localhost:5672/%2f/completeness
```

Comment:
Require changes in docker repository

Reply:
PR https://github.com/ergonode/docker/pull/44